golang/scripts/build.sh

```diff
@@ -0,0 +1,20 @@
+#!/usr/bin/env bash
+# ^^^^^^^^^^^^^^^^^ this is the most platform-agnostic way to guarantee this script runs with Bash
+# 2021-07-08 WATERMARK, DO NOT REMOVE - This script was generated from the Kurtosis Bash script template
+
+set -euo pipefail   # Bash "strict mode"
+script_dirpath="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+root_dirpath="$(dirname "${script_dirpath}")"
+
+
+# ==================================================================================================
+#                                             Constants
+# ==================================================================================================
+KURTOSIS_DOCKERHUB_ORG="kurtosistech"
+IMAGE_NAME="kurtosis-lambda-starter-pack"
```

Comment:
Note: this will need to change during bootstrapping (in the same way that we do for testsuites)